save-common/src/jvmMain/kotlin/org/cqfn/save/entities/AgentStatus.kt

```diff
@@ -6,21 +6,28 @@ import javax.persistence.Column
 import javax.persistence.Entity
 import javax.persistence.EnumType
 import javax.persistence.Enumerated
-import javax.persistence.GeneratedValue
-import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+import javax.persistence.Table
 
 /**
  * @property time
  * @property state
- * @property agentId
- * @property id
+ * @property agent
  */
 @Entity
+@Table(name = "agent_status")
```

Comment:
Why have you added explicit names for tables and columns?

Reply:
I think this makes the classes more readable and maintains the same style :)

Reply:
I see your point, but now it mostly means duplication of class and field names. I think we can use explicit annotations only when name of the column differs from field name or when special relationship exists between tables.